drivers/hmis_external_apis/app/models/hmis_external_apis/ac_hmis/importers/housing_assessment_importer.rb

```diff
@@ -236,17 +236,30 @@ def ensure_intake_assessment!(waitlist, enrollment)
     ActiveRecord::Base.record_timestamps = true
   end
 
-  # TODO: if not found by MCI Unique ID, should we try to look up by MCI ID?
   def find_and_update_hmis_client(waitlist)
```

Comment:
note, this method isn't updating the client anymore so the method name isn't accurate